src/main/java/com/google/codeu/servlets/MessageServlet.java

```diff
@@ -77,10 +77,11 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
 
     String user = userService.getCurrentUser().getEmail();
     String text = Jsoup.clean(request.getParameter("text"), Whitelist.none());
+    String recipient = request.getParameter("recipient");
```

Comment:
I am not familiar with the Serverlet so this may be a silly question, my apologies. How does this parameter gets in the request?

Reply:
From my understanding I think "recipient" is the key and whatever the actual recipient is would be the value, and getParameter would get it for that specific entry? The guide said this shouldn't actually do anything yet.